pkg/virtctl/vm/common.go

```diff
@@ -48,6 +49,7 @@ var (
 	volumeName   string
 	persist      bool
 	dryRun       bool
+	dryRunOption []string
```

Comment:
The usage of globals is discouraged.
It is unfortunate that we have many in-place already, but at least we better not add more.

Reply:
Updated to eliminate the global variable.